README.md

```diff
@@ -60,7 +60,7 @@ Dor::Services:Client provides a number of methods to simplify connecting to the
 objects_client = Dor::Services::Client.objects
 
 # Register a non-existent object
-objects_client.register(params: {})
+objects_client.register(params: {}, assign_doi: false)
```

Comment:
How about leaving this as it was (since false is the default) and then adding a new line that shows how to register with `assign_doi: true`

Reply:
Fixed.